### PR TITLE
Move scheduler overview from docs/design/ to docs/devel/

### PR DESCRIPTION
**Pull request by davidopp**
Per our conversation yesterday.

@thockin  @erictune

### CODE REVIEW

File: docs/devel/scheduler.md [hunk 1]
```diff
@@ -48,3 +48,6 @@ If you want to get a global picture of how the scheduler works, you can start in
 
 
 [![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/docs/design/scheduler.md?pixel)]()
```
Comment (mikedanese) [inline docs/devel/scheduler.md]: remove the ga-tracer from the last location

Comment (davidopp) [inline docs/devel/scheduler.md]: done, thanks for noticing

**Comment by mikedanese** [comment]
LGTM

**Comment by thockin** [comment]
LGTM

**Pull request merged**